### PR TITLE
displaying response information to the user after results have been submitted

### PR DESCRIPTION
**Pull request by acornett21**
- fixes: #376 
- displaying the image id to the user and then showing them a URL to the project overview.

Signed-off-by: Adam D. Cornett <adc@redhat.com>

### CODE REVIEW

**Comment by acornett21** [comment]
@tonytcampbell Is along the lines of the message that we want to show to the user?

File: cmd/check_container.go [hunk 1]
```diff
@@ -191,11 +191,15 @@ var checkContainerCmd = &cobra.Command{
...
 			}
 
-			// TODO: use the return values once we know what we need to display to the user
-			_, _, _, err = pyxisEngine.SubmitResults(certProject, certImage, rpmManifest, testResults)
+			_, certImage, _, err = pyxisEngine.SubmitResults(certProject, certImage, rpmManifest, testResults)
 			if err != nil {
 				return err
 			}
+
+			log.Infof("Test results have been submitted to Red Hat."+
```
Comment (bcrochet) [inline cmd/check_container.go:199]: These might be better as individual log statements?

Comment (acornett21) [reply cmd/check_container.go:199]: I was thinking that as well, or was wondering if there was a way to formate like a `paragraph` for a log message? But I don't think that's a thing, even `\n` shows up as a literal.

**Review by jomkz — APPROVED** [review]
/lgtm

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**